src/System.CommandLine/CommandLineConfiguration.cs

```diff
@@ -22,7 +22,7 @@ public class CommandLineConfiguration
         /// <summary>
         /// Initializes a new instance of the CommandLineConfiguration class.
         /// </summary>
-        /// <param name="symbol">The symbol to parse.</param>
+        /// <param name="command">The symbol to parse.</param>
```

Comment:
```suggestion
        /// <param name="command">The command to parse.</param>
```

Reply:
Oops, missed this comment. I'll do it in a different PR.